plone/app/iterate/subscribers/workflow.py

```diff
@@ -47,7 +47,7 @@ def handleCheckout(event):
     settings = registry.forInterface(IIterateSettings)
     if not settings.enable_checkout_workflow:
         return
-    policy_id = str(settings.checkout_workflow_policy)
```

Comment:
the str() is needed because Products.CMFWorkflowPolicy really needs that to be a string.